cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/services/ProgramLifecycleService.java

```diff
@@ -159,6 +159,31 @@ public ProgramStatus getProgramStatus(ProgramId programId) throws Exception {
     return getExistingAppProgramStatus(appSpec, programId);
   }
 
+  /**
+   * Returns the program run count of the given program.
+   *
+   * @param programId the id of the program for which the count call is made
+   * @return the run count of the program
+   * @throws NotFoundException if the application to which this program belongs was not found or the program is not
+   *                           found in the app
+   */
+  public int getProgramRuncount(ProgramId programId) throws Exception {
```

Comment:
"run count" are two words, so you should use `RunCount` in code.

Reply:
fixed